website/content/en/docs/golang/metrics/operator-sdk-monitoring.md

```diff
@@ -87,6 +87,76 @@ By default operator will expose info metrics based on the number of the current
 
 The operator uses [Prometheus][prometheus] to expose a number of metrics by default. In order to expose custom metrics they have to be registered with the `Registry` object. An example can be found in the [kubebuilder book][kubebuilder].
 
+### With 3rd party resource
```

Comment:
Is this related to Prometheus?

Reply:
Are you asking if `3rd party resources` are related to Prometheus? 
No. 3rd party resources are any resource which is not from Kubernetes API.

Reply:
Was added further information here. Please, let us know if it is clear for you now.

Reply:
The reason I asked is that your other PR changes the title of this file. https://github.com/operator-framework/operator-sdk/pull/2821#pullrequestreview-391592131

If this isn't related to Prometheus, I dont think this belongs here.

Reply:
The metrics are exported to Prometheus. If you check this doc you will see that it is speaking over this specific method as well above.